src/pages/lesson-details/LessonDetails.tsx

```diff
@@ -0,0 +1,103 @@
+import { useCallback, useState } from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { AxiosResponse } from 'axios'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+import useAxios from '~/hooks/use-axios'
+import { ResourceService } from '~/services/resource-service'
+import {
+  attachmentsMock,
+  defaultResponse
+} from '~/pages/lesson-details/LessonDetails.constants'
+import Accordions from '~/components/accordion/Accordions'
+import IconExtensionWithTitle from '~/components/icon-extension-with-title/IconExtensionWithTitle'
+
+import { errorRoutes } from '~/router/constants/errorRoutes'
+import { styles } from '~/pages/lesson-details/LessonsDetails.styles'
+import { Lesson, TypographyVariantEnum } from '~/types'
+import Loader from '~/components/loader/Loader'
+
+const LessonDetails = () => {
+  const [activeItems, setActiveItems] = useState<number[]>([])
+
+  const { id } = useParams()
+  const navigate = useNavigate()
+
+  const onChange = (activeItem: number) => {
+    setActiveItems((prevActiveItems) => {
+      if (prevActiveItems.includes(activeItem)) {
+        return prevActiveItems.filter(
+          (prevActiveItem) => prevActiveItem !== activeItem
```

Comment:
can you just return prevActiveItems or [...prevActiveItems] here?

Reply:
If do it I will cannot close the accordion.